packages/sanity/src/core/form/members/object/fields/ArrayOfObjectsField.tsx

```diff
@@ -90,9 +91,25 @@ export function ArrayOfObjectsField(props: {
 
   const handleChange = useCallback(
     (event: PatchEvent | PatchArg) => {
+      const patches = PatchEvent.from(event).patches
+      // if the patch is an unset patch that targets an item in the array (as opposed to unsetting a field somewhere deeper)
+      const hasUnset = patches.some((patch) => patch.type === 'unset' && patch.path.length === 1)
```

Comment:
Re-reading this I realize that it would be more accurate to name this `isRemovingItem` or similar, given that it reflects whether an item is removed from the array, not whether there is an unset patch in the array.

Reply:
named it `isRemovingLastItem` let me know if that makes sense